Modules/_winapi.c

```diff
@@ -1911,23 +1910,23 @@ static PyMethodDef winapi_functions[] = {
 };
 
 #define WINAPI_CONSTANT(fmt, con) \
-    PyDict_SetItemString(d, #con, Py_BuildValue(fmt, con))
+    if (PyDict_SetItemString(d, #con, Py_BuildValue(fmt, con)) < 0) return -1
```

Comment:
Hum, this code leaks a reference. Py_BuildValue() creates a new reference, but PyDict_SetItemString() also creates a new reference. A Py_DECREF() is needed somewhere.